src/gps_spoofer.py

```diff
@@ -86,6 +118,10 @@ def publish_odometry(self, event):
     self.pwc.pose.pose.position.y += self.R_noise*sin(self.theta_noise)
```

Comment:
Not part of this PR, but please rename this method from 'publish_odometry' to something like 'publish_noisy_positions'.